### DISCUSSION TITLE
Updating Dockerfiles to use python 3.10 is too aggressive

### DISCUSSION THREAD

**Discussion by zcutlip** [renovatebot/renovate#13611] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

I have a suite of Dockerfiles I use for testing against a set of Python versions. Renovatebot created a PR to upgrade them to use python 3.10.

E.g.
```diff
@@ -1,4 +1,4 @@  
- FROM python:3.8
+ FROM python:3.10
```

I'm sure it's a common Docker use case to test against an assortment of Python versions. And the versions I'm testing against are still receiving official updates.

Here's the PR in question:
https://github.com/zcutlip/pyonepassword/pull/27/files

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment, marked as answer]
This not a bug, so I've converted it to a discussion. Please start with a discussion first in future.

It's Renovate's job to update dependencies, and it's doing exactly what it should. If you have intentionally "behind" dependencies such as for testing, you can either:
- Disable Renovate on those folders altogether, or
- Use `packageRules` + `allowedVersions` to restrict the ranges allowed to update to
